boa/src/builtins/object/gcobject.rs

```diff
@@ -0,0 +1,309 @@
+//! This module implements the `GcObject` structure.
```

Comment:
We should remove this file, it seems to be a duplicate of ` boa/src/object/gcobject.rs` and was move in a previous commit